### DISCUSSION TITLE
jommcrm and joommedia

### DISCUSSION THREAD

**Discussion by spinelbusiness** [JoomCoder-com/JoomCCK#32] [Ideas]
hello
really good news, someone is rebuilding cobalt. i  wanted to buy 20 licences for my compagny but asmuz never replied me.
hope an integration with joomcrm (of course :-) joom video joom project. cobalt was an amazing tool especially forum features (it was the only one forum with autocomplete field, relate child/parent fields, ect) but it nned some integrations. thank you

**Comment by Giorgi625** [comment]
I think you will not need separate licences, it should not be limited to single domain/ You get subscription and use it on multiple domains without problem.

**Reply by andiamine** [reply]
Hi my friend, yes we are going to use a subscription system.
